core/src/main/java/org/apache/spark/unsafe/map/BytesToBytesMap.java

```diff
@@ -53,7 +53,8 @@
  * The map can support up to 2^29 keys. If the key cardinality is higher than this, you should
  * probably be using sorting instead of hashing for better cache locality.
  *
- * The key and values under the hood are stored together, in the following format:
+ * The key and values under the hood are stored together, in the following format(in case of
+ * uaoSize = 4):
```

Comment:
why not replace the number with `uaoSize`?

Reply:
I was supposed to do so but it requires more changes in the original content and it is not so clean compares to a concrete number. So I am not sure if we should do. Do you have strong prefer to change in `uaoSize` way?

Reply:
The current way doesn't make it clear whether each number `4` represents the uaoSize.

Reply:
Ok. I'll update later.